charts/seed-controlplane/charts/kube-controller-manager/templates/kube-controller-manager.yaml

```diff
@@ -77,7 +77,10 @@ spec:
         - --concurrent-replicaset-syncs=10
         {{- include "kube-controller-manager.featureGates" . | trimSuffix "," | indent 8 }}
         - --horizontal-pod-autoscaler-downscale-delay={{ .Values.horizontalPodAutoscaler.downscaleDelay }}
+        - --horizontal-pod-autoscaler-sync-period={{ .Values.horizontalPodAutoscaler.syncPeriod }}
+        - --horizontal-pod-autoscaler-tolerance={{ .Values.horizontalPodAutoscaler.tolerance }}
         - --horizontal-pod-autoscaler-upscale-delay={{ .Values.horizontalPodAutoscaler.upscaleDelay }}
+        - --horizontal-pod-autoscaler-use-rest-clients={{ .Values.horizontalPodAutoscaler.useRestClients }}
```

Comment:
Let's not expose this flag, I can't think of a scenario where it makes sense to set this to false. Better, let's not even specify it in our KCM flag config.

Reply:
FYI: It doesn't even exist any more in 1.12, so let's not even introduce a deprecated/to-be-removed flag.

Reply:
Agreed. I will remove it.

Reply:
Removed it. Even from the helm chart parameters!